sebs.py

```
@@ -307,6 +307,51 @@ def process(**kwargs):
+        sebs_client.config.image_tag_prefix = image_tag_prefix
+
+    experiment_config = sebs_client.get_experiment_config(config["experiments"])
+    update_nested_dict(config, ["experiments", "benchmark"], benchmark)
+    benchmark_obj = sebs_client.get_benchmark(
+        benchmark,
+        deployment_client,
+        experiment_config,
+        logging_filename=logging_filename,
+    )
+
+    func = deployment_client.build_function(
```

Ruff (0.8.2)

Local variable `func` is assigned to but never used

Remove assignment to unused variable `func`

(F841)